model/publishing/delivery/PublishingDeliveryService.php

```diff
@@ -51,140 +40,4 @@ class PublishingDeliveryService extends ConfigurableService
     const DELIVERY_REMOTE_SYNC_COMPILE_ENABLED = 'http://www.tao.lu/Ontologies/TAODelivery.rdf#ComplyEnabled';
 
     const DELIVERY_REMOTE_SYNC_REST_OPTION = 'remote-publish';
```

Comment:
Nice clean up! We should maybe create a tech task for moving the constant  to the `\oat\taoPublishing\model\publishing\delivery\RemotePublishingService` along with service definition and external usages

Reply:
That would be nice. I have local branch where I started to remove all this code, we can include this change there as well.